Index: lua-5.3.4/src/lmathlib.c
===================================================================
--- lua-5.3.4.orig/src/lmathlib.c
+++ lua-5.3.4/src/lmathlib.c
@@ -183,7 +183,7 @@ static int math_log (lua_State *L) {
     res = l_mathop(log)(x);
   else {
     lua_Number base = luaL_checknumber(L, 2);
-#if !defined(LUA_USE_C89)
+#if !defined(LUA_USE_C89) && !defined(__BIONIC__)
     if (base == l_mathop(2.0))
       res = l_mathop(log2)(x); else
 #endif
