diff -Nuar SDL2-2.0.8.orig/include/SDL_config_iphoneos.h SDL2-2.0.8/include/SDL_config_iphoneos.h
--- SDL2-2.0.8.orig/include/SDL_config_iphoneos.h	2018-03-01 17:34:41.000000000 +0100
+++ SDL2-2.0.8/include/SDL_config_iphoneos.h	2018-03-17 19:26:49.000000000 +0100
@@ -157,7 +157,8 @@
 #define SDL_VIDEO_RENDER_OGL_ES2    1
 
 /* Metal supported on 64-bit devices running iOS 8.0 and tvOS 9.0 and newer */
-#if !TARGET_OS_SIMULATOR && !TARGET_CPU_ARM && ((__IPHONE_OS_VERSION_MIN_REQUIRED >= 80000) || (__TV_OS_VERSION_MIN_REQUIRED >= 90000))
+//#if !TARGET_OS_SIMULATOR && !TARGET_CPU_ARM && ((__IPHONE_OS_VERSION_MIN_REQUIRED >= 80000) || (__TV_OS_VERSION_MIN_REQUIRED >= 90000))
+#if 0
 #define SDL_PLATFORM_SUPPORTS_METAL	1
 #else
 #define SDL_PLATFORM_SUPPORTS_METAL	0
