Index: boost_1_75_0/boost/geometry/strategies/cartesian/buffer_end_round.hpp
===================================================================
--- boost_1_75_0.orig/boost/geometry/strategies/cartesian/buffer_end_round.hpp
+++ boost_1_75_0/boost/geometry/strategies/cartesian/buffer_end_round.hpp
@@ -24,7 +24,7 @@
 #include <boost/geometry/strategies/buffer.hpp>
 
 
-#include <boost/geometry/io/wkt/wkt.hpp>
+//#include <boost/geometry/io/wkt/wkt.hpp>
 
 namespace boost { namespace geometry
 {
Index: boost_1_75_0/boost/math/constants/calculate_constants.hpp
===================================================================
--- boost_1_75_0.orig/boost/math/constants/calculate_constants.hpp
+++ boost_1_75_0/boost/math/constants/calculate_constants.hpp
@@ -786,6 +786,7 @@ inline T constant_extreme_value_skewness
   return ev;
 }
 
+#if 0
 namespace detail{
 //
 // Calculation of the Glaisher constant depends upon calculating the
@@ -907,6 +908,7 @@ inline T constant_glaisher<T>::compute(B
   return g;
   */
 }
+#endif
 
 template <class T>
 template<int N>
