Index: tiff-4.4.0/CMakeLists.txt
===================================================================
--- tiff-4.4.0.orig/CMakeLists.txt
+++ tiff-4.4.0/CMakeLists.txt
@@ -135,12 +135,6 @@ include(Release)
 # Process subdirectories
 add_subdirectory(port)
 add_subdirectory(libtiff)
-add_subdirectory(tools)
-add_subdirectory(test)
-add_subdirectory(contrib)
-add_subdirectory(build)
-add_subdirectory(man)
-add_subdirectory(html)
 
 # pkg-config support
 include(PkgConfig)
