Index: tiff-4.0.10/CMakeLists.txt
===================================================================
--- tiff-4.0.10.orig/CMakeLists.txt
+++ tiff-4.0.10/CMakeLists.txt
@@ -711,12 +711,6 @@ endif()
 # Process subdirectories
 add_subdirectory(port)
 add_subdirectory(libtiff)
-add_subdirectory(tools)
-add_subdirectory(test)
-add_subdirectory(contrib)
-add_subdirectory(build)
-add_subdirectory(man)
-add_subdirectory(html)
 
 #message(STATUS "EXTRA_DIST: ${EXTRA_DIST}")
 
